tests/contrib/langchain/test_langchain_community.py

```diff
@@ -1274,3 +1274,65 @@ def test_faiss_vectorstore_retrieval(langchain_community, langchain_openai, requ
             retriever = faiss.as_retriever()
         with request_vcr.use_cassette("openai_retrieval_embedding.yaml"):
             retriever.invoke("What was the message of the last test query?")
+
+
+@pytest.mark.snapshot(
+    ignores=["meta.langchain.request.tool.description"],
```

Comment:
Why do we need to ignore the tool description tag in the snapshot test?

Reply:
Due to some subtle differences in langchain core versions the description might be different.
```
"Circumference calculator(radius: float) -> float - Use this tool when you need to calculate a circumference using the radius of a circle"
```
In the other versions it's just 
```
"Use this tool when you need to calculate a circumference using the radius of a circle"
```

So instead of using two cassettes for each case i just ignored the field

Reply:
Ahh I see. Thanks for clarifying, let's add a small comment next to the ignore line mentioning this!